modules/fedimint-lnv2-client/src/events.rs

```diff
@@ -0,0 +1,65 @@
+use fedimint_core::Amount;
+use fedimint_core::core::{ModuleKind, OperationId};
+use fedimint_eventlog::{Event, EventKind, EventPersistence};
+use serde::{Deserialize, Serialize};
+
+use crate::LightningInvoice;
+
+/// Event emitted when a send operation is created.
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+pub struct SendEvent {
+    pub operation_id: OperationId,
+    pub amount: Amount,
+    pub invoice: Option<LightningInvoice>,
+}
+
+impl Event for SendEvent {
+    const MODULE: Option<ModuleKind> = Some(fedimint_lnv2_common::KIND);
+    const KIND: EventKind = EventKind::from_static("send");
+    const PERSISTENCE: EventPersistence = EventPersistence::Persistent;
+}
+
+/// Status of a send operation.
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq)]
+pub enum SendStatus {
```

Comment:
Should we use internally tagged enum serialization? https://serde.rs/enum-representations.html . I generally find it preferable and most natural.